dora/build.rs

```diff
@@ -0,0 +1,51 @@
+use std::{
+    env, error::Error, fs::{self, File, copy}, io::Write, path::Path,
+};
+
+const SOURCE_DIR: &str = "stdlib";
+
+fn main() -> Result<(), Box<dyn Error>> {
+    let out_dir = env::var("OUT_DIR")?;
+    let out_path = Path::new(&out_dir);
+
+    let root_dir = env::var("CARGO_MANIFEST_DIR")?;
+    let root_path = Path::new(&root_dir);
+
+    let copy_path = out_path.join("stdlib");
+    let stdlib_path = out_path.join("stdlib.rs");
+
+    if copy_path.is_dir() {
+        fs::remove_dir_all(&copy_path)?;
+        fs::create_dir(&copy_path)?;
+    } else {
+        fs::create_dir(&copy_path)?;
+    }
+
+    if stdlib_path.is_file() {
+        fs::remove_file(&stdlib_path)?;
+    }
+
+    let mut stdlib = File::create(&stdlib_path)?;
+
+    writeln!(&mut stdlib, r#"["#,)?;
+
+    for f in fs::read_dir(SOURCE_DIR)? {
+        let f = f?;
+
+        if !f.file_type()?.is_file() {
+            continue;
+        }
+
```

Comment:
I think I should add a guard, that the to-be-included-files end with `.dora`

Reply:
Yes, I agree. Out of interest: Does this also include files recursively?

Reply:
Yes, please add this flag.

Reply:
> Yes, I agree. Out of interest: Does this also include files recursively?

No, it does not. I can add it in a different PR

Reply:
I added a check now